tasks/cockpit-tasks

```diff
@@ -113,19 +113,41 @@ function run_one_task() {
     fi
 }
 
+function run_from_queue() {
+    cd "$XDG_CACHE_HOME"/cockpit-project/bots
+    # Consuming directly from queue, commands generated by run-queue,
+    # tests-scan, and issue-scan have bots/ prepended
+    [ -e bots ] || ln -s . bots
+    if /usr/bin/timeout 12h /bin/sh -exc "./run-queue --amqp \"$AMQP_SERVER\""; then
```

Comment:
What does the extra `sh -exc` actually give you, compared to just directly running run-queue in timeout?

Reply:
oh good point, I read the previous comment wrong.

Reply:
That's fine, I wrote it wrongly in the first place :-)